hack/update-codegen.sh

```diff
@@ -22,16 +22,16 @@ SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 CODEGEN_PKG=${CODEGEN_PKG:-$(cd "${SCRIPT_ROOT}"; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ../code-generator)}
 
 bash "${CODEGEN_PKG}"/generate-internal-groups.sh \
-  "deepcopy,conversion" \
+  "deepcopy,conversion,defaulter" \
   sigs.k8s.io/scheduler-plugins/pkg/generated \
-  sigs.k8s.io/scheduler-plugins/pkg/apis \
-  sigs.k8s.io/scheduler-plugins/pkg/apis \
+  sigs.k8s.io/scheduler-plugins/apis \
+  sigs.k8s.io/scheduler-plugins/apis \
   "config:v1beta2,v1beta3" \
   --go-header-file "${SCRIPT_ROOT}"/hack/boilerplate/boilerplate.generatego.txt
 
 bash "${CODEGEN_PKG}"/generate-groups.sh \
   all \
   sigs.k8s.io/scheduler-plugins/pkg/generated \
-  sigs.k8s.io/scheduler-plugins/pkg/apis \
+  sigs.k8s.io/scheduler-plugins/apis \
```

Comment:
Given we're going to use kubebuilder to cover the codegen for apis/scheudling/*, we can remove L32~L37. As a replacement, let's place the controller-gen script here.